src/scripts/prepare-prod.js

```diff
@@ -36,32 +34,17 @@ const hostingStrategy = process.env.HOSTING_STRATEGY || "custom";
         hostingStrategy
       )}\n`
     );
-
-    // create a netlify folder at the root
-    execa("mkdir", ["-p", "netlify"], {
-      stdio: "inherit",
-    });
-
-    // Copying the netlify folder to the root directory
-    execa(
-      "cp",
-      [
-        "-r",
-        "node_modules/rasengan/lib/esm/server/functions/netlify/functions",
-        "./netlify",
-      ],
-      {
-        stdio: "inherit",
-      }
-    );
-
-    // Copying the netlify.toml file to the root directory
+    // Copying the netlify folder content to the root directory
     execa(
-      "cp",
-      ["node_modules/rasengan/lib/esm/server/functions/netlify/netlify.toml", "."],
+      copyCommand,
+      generateCopyExecaArray(
+        "node_modules/rasengan/lib/esm/server/functions/netlify",
+        "."
+      ),
```

Comment:
There is a problem here.

This command will only copy the content of the netlify folder, so at the destination, you will have the "functions" directory instead of the "netlify" one.

Here is what the final result has to be 👇🏼
<img width="251" alt="Screenshot 2024-09-12 at 14 06 24" src="https://github.com/user-attachments/assets/0395c704-1746-46cb-8a17-942b74baf8b5">

You can see, there are:
- netlify/functions/api.js
- netlify.toml

It's the expected result.

Reply:
Yes I updated the structure of the server/functions/netlify for it

Reply:
Good good ✨